lib/electric_slide/call_queue.rb

```diff
@@ -276,11 +276,9 @@ def bridge_agent(agent, queued_call)
         remove_agent agent
       end
 
-      agent.call.on_joined do 
-        agent.callback :connect, self, agent.call, queued_call
-      end
-
       agent.join queued_call
+      agent.callback :connect, self, agent.call, queued_call
```

Comment:
the idea is that it will be used for a screen-pop.  Since we want that to be real-time, I think we probably want this to fire when the agent is selected.